src/function/scalar/list/list_reverse.cpp

```diff
@@ -40,19 +40,15 @@ static void ListReverseFunction(DataChunk &args, ExpressionState &state, Vector
 	for (idx_t i = 0; i < count; i++) {
 		auto input_list_list_index = input_list_data.sel->get_index(i);
 
-		if (!input_list_data.validity.RowIsValid(input_list_list_index)) {
```

Comment:
That might be a little too eager of a change, no?
The comment was about removing the check after this, *because* this check ensures that it's always true

Unless this is not needed because of something else that I'm not following?

Reply:
Hello @Tishj,
Thanks again for the fast feedback. I may have cleaned up a bit too much. I wasn't running into any issues for some reason. I will add another test for mistakes in the validity map. Thank you again!